app/controllers/coronavirus_local_restrictions_controller.rb

```diff
@@ -16,11 +16,11 @@ def results
       return render_no_postcode_error
     end
 
-    @postcode = params["postcode-lookup"].gsub(/\s+/, "").upcase
+    @postcode = PostcodeService.new(params["postcode-lookup"]).sanitize
 
     if @postcode.blank?
       return render_no_postcode_error
-    elsif !postcode_validation
+    elsif !PostcodeService.new(@postcode).valid?
```

Comment:
Hey according to the library (https://github.com/threedaymonk/uk_postcode) partial postcodes are valid, is this fine?

Reply:
ignore - I can see you're using the same regex.

Should we be testing for partial postcodes?

Reply:
It might be a nice route to handle, but OK to defer for now? On local transactions I think we tracked the different sorts of errors, like valid postcode but no location associated with it, partial postcode, not a postcode, those sorts of things.

Reply:
I can add a test to check partial postcodes fail validation.